docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -25,22 +25,68 @@ NOTE: Most of the existing CDI code should work just fine but there are some sma
 == Bean Discovery
 
 Bean discovery in CDI is a complex process which involves legacy deployment structures and accessibility requirements of the underlying module architecture.
-Quarkus is using a simplified bean discovery.
-There is only one bean archive with `annotated` bean discovery mode and no visibility boundaries.
+However, Quarkus is using a *simplified bean discovery*.
```

Comment:
I would rather have it say `Quarkus uses`, but let's not hold up the PR for something so simple